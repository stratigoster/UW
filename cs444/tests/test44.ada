PACKAGE foo IS
  type colors is (white,neon,black);
  type pos is array (1..(2**32)-1, integer range <>, colors) of boolean;
--                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-- array is either constrained xor unconstrained
END foo;
