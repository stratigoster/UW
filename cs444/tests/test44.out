***ERROR***:3:73:	Mixing unconstrained and constrained array definitions

------
Compilation error
------
